### ISSUE TITLE
sdk formats objects from AST weirdly

### ISSUE THREAD

**Issue by smacker** [bblfsh/sdk#214]
AST -> UAST
```
{"langSpecific": {"a": 1}} -> {"Properties": {"langSpecific": "map[a:1]"}}
```

Such format isn't very useful.

more info here: https://github.com/bblfsh/sdk/pull/213#issuecomment-347505198

**Comment by juanjux** [comment]
I see two cases here:

- The property is semantically unimportant. Then it doesn't matter what format it really have, trough mapping objects and arrays to the JSON format would be nice.

- The property is semantically important. Then it should be converted to a UAST node (or list of nodes) with roles and such using the promote* config or one of the ToNoder callbacks.

**Comment by smacker** [comment]
Thank you for your opinion!

Then I see 2 possible improvements here:

- Reflect need of using promote* or modifier (in case of array) for semantically important properties here: https://doc.bblf.sh/driver/annotations.html 
- Format as JSON to make it nicer.

I'll send PR for the latter later.

**Comment by juanjux** [comment]
Fixed by #215.
